Library/Homebrew/cmd/test-bot.rb

```diff
@@ -112,33 +112,47 @@ def run
         return
       end
 
+      @output = ""
       start_time = Time.now
+      verbose = ARGV.verbose?
+      working_dir = Pathname.new(@command.first == "git" ? @repository : Dir.pwd)
+      puts if verbose
 
-      log = log_file_path
+      File.open(log_file_path, "wb") do |log|
```

Comment:
I'd perhaps consider (as I mentioned before) only opening the file if there's a non-empty string to write out.